app/components/HomeComp.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState, useEffect } from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import { getbannersListDetails, getCategoriesListDetails } from '../api/api';
+import { BANNER_ERROR_MSG, CATEGORIES_ERROR_MSG, ERROR } from '../constant';
+import { images } from './common/Common';
+import Swal from 'sweetalert2';
+
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+
+
+const HomeComp = () => {
+    const [bannersObject, setBannersObject] = useState([]);
+    const [productsObject, setProductsObject] = useState([]);
+
+    /**
+     * Similar to componentDidMount and componentDidUpdate:
+     */
+    useEffect(() => {
+        bannerGetCall();
+        productGetCall();
+    }, []);
+
+    /**
+    * function for banners list data
+    */
+   const successCallback = response => {
+        setBannersObject(
+            ...bannersObject,
+            response.data
+        )
+   }
+   const failureCallback = error => {
+        Swal.fire(BANNER_ERROR_MSG, response.message, ERROR)
+   }
+    const bannerGetCall = async () => {
+        getbannersListDetails(successCallback,failureCallback)
+        // const response = await getbannersListDetails()
+        // if (response && response.status === 200 && response.data) {
+        //     setBannersObject(
+        //         ...bannersObject,
+        //         response.data
+        //     )
+        // }
+        // else {
+        //     Swal.fire(BANNER_ERROR_MSG, response.message, ERROR)
+        // }
+    }
+    /**
+    * function for products list Data
+    */
+    const productGetCall = async () => {
+        const response = await getCategoriesListDetails();
+        if (response && response.status === 200 && response.data) {
+            setProductsObject(
+                ...productsObject,
+                response.data
+            )
+        }
```

Comment:
use the api call the same way as at line no 36

Reply:
Done.